README.md

```diff
@@ -27,5 +27,8 @@ Reach out to us on [Slack](https://www.pokt.network/slack-pokt) and start a disc
 - Code must be documented adhering to the official Go commentary guidelines.
 - Pull requests need to be based on and opened against the `staging` branch.
 
+# How to build
+go build pokt-network/pocket-core/cmd/pocket_core/main.go
```

Comment:
Maybe a little more explanation? Like where to find the executables and also the formatting, for commands use backtick:

`go build pokt-network/pocket-core/cmd/pocket_core/main.go`

Reply:
The executable location will be set in the go config so we can't really document that

Reply:
Got it, thanks.